hetio/neo4j.py

```diff
@@ -8,14 +8,28 @@
 import py2neo.packages.httpstream
 import pandas
 
+from tqdm import tqdm
+from functools import reduce
+from operator import or_
+
+PY2NEO_VER = int(py2neo.__version__[0])
+
 import hetio.hetnet
+import hetio.stats
 
 # Avoid SocketError
 py2neo.packages.httpstream.http.socket_timeout = 1e8
 
-def export_neo4j(graph, uri, node_queue=100, edge_queue=100):
+def export_neo4j(graph, uri, node_queue=100, edge_queue=100, show_progress=False):
     """Export hetnet to neo4j"""
-    db_graph = py2neo.Graph(uri)
+
+    if PY2NEO_VER == 2:
```

Comment:
Sure, that's an easy fix I can make. I guess it might just confuse other people if they try to use the py2neo 3 URI resolver with bolt and have difficulties with getting it working.